pkg/render/logstorage.go

```diff
@@ -653,7 +687,7 @@ func (es elasticsearchComponent) eckOperatorStatefulSet() *appsv1.StatefulSet {
 					Containers: []corev1.Container{{
 						Image: components.GetReference(components.ComponentElasticsearchOperator, es.installation.Spec.Registry, es.installation.Spec.ImagePath),
 						Name:  "manager",
-						Args:  []string{"manager", "--operator-roles", "all", "--enable-debug-logs=false"},
+						Args:  []string{"manager", "--operator-roles", "all", "--log-verbosity=2"},
```

Comment:
What is level 2 equivalent to? Info?

Reply:
Thanks for reminding me, it should be 0.


> Verbosity level of logs. -2=Error, -1=Warn, 0=Info, 0 and above=Debug